prow/tide/tide_test.go

```diff
@@ -1582,6 +1589,123 @@ func TestSync(t *testing.T) {
 	}
 }
 
+func TestIsPassing(t *testing.T) {
+	yes := true
+	no := false
+	headSHA := "head"
+	success := string(githubql.StatusStateSuccess)
+	failure := string(githubql.StatusStateFailure)
+	testCases := []struct {
+		name             string
+		passing          bool
+		optional         []string
+		config           *config.Policy
+		combinedContexts map[string]string
+	}{
+		{
+			name:    "required checks disabled should not impact - passing",
+			passing: true,
+			config: &config.Policy{
```

Comment:
@fejta is it even valid config to have a policy with explicitly listed required checks but have the policy be set to not enforce? why?

Reply:
I think there could multiple use cases. When you define policy at the repo or org level, and then you create a branch for testing and you don't want to protect it or if you just want to disable temporarily.